cmd/architect/architect.go

```diff
@@ -35,13 +35,36 @@ func RunArchitect(configuration RunConfiguration) {
 		logrus.Fatalf("Could not parse registry credentials %s", err)
 	}
 
+	var builder process.Builder
+
+	if strings.Contains(strings.ToLower(c.BuildStrategy), config.Buildah) {
+		logrus.Info("ALPHA FEATURE: Running buildah builds")
+		builder = &process.BuildahCmd{
+			TlsVerify: c.TlsVerify,
+		}
+
+	} else {
+		if !strings.Contains(c.BuildStrategy, config.Docker) {
+			logrus.Warnf("Unsupported build strategy: %s. Defaulting to docker", c.BuildStrategy)
+		}
+
+		builder, err = process.NewDockerBuilder()
```

Comment:
En liten tanke, vi kunne pakket inn det hele med en ny BuilderClient funksjon og flyttet mye av koden over ut.

Reply:
Det høres ut som en litt større oppgave.  Opprette en jira-task for det?